NitroxModel/Helper/NitroxUser.cs

```diff
@@ -78,7 +82,9 @@ public static string GamePath
                 }
 
                 List<string> errors = new();
-                string path = GameInstallationFinder.Instance.FindGame(errors);
+                GameInstallation installation = GameInstallationFinder.Instance.FindGame(GameInfo.Subnautica, GameLibraries.ALL, errors).First();
+                string path = installation.Path;
+
                 if (!string.IsNullOrWhiteSpace(path) && Directory.Exists(path))
```

Comment:
`Directory.Exists()` already checks for null and empty string won't be a valid path. See image below:
```suggestion
                if (Directory.Exists(path))
```
![image](https://github.com/SubnauticaNitrox/Nitrox/assets/1107063/89a83747-1ede-4d54-a68e-0ac14f8f7d09)